elements/lisk-p2p/src/p2p.ts

```diff
@@ -390,6 +394,18 @@ export class P2P extends EventEmitter {
 			outboundShuffleInterval: config.outboundShuffleInterval
 				? config.outboundShuffleInterval
 				: DEFAULT_OUTBOUND_SHUFFLE_INTERVAL,
+			latencyProtectionRatio:
+				typeof config.latencyProtectionRatio === 'number'
+					? config.latencyProtectionRatio
+					: DEFAULT_PEER_PROTECTION_FOR_LATENCY_AND_USEFULNESS,
+			productivityProtectionRatio:
+				typeof config.productivityProtectionRatio === 'number'
+					? config.productivityProtectionRatio
+					: DEFAULT_PEER_PROTECTION_FOR_LATENCY_AND_USEFULNESS,
```

Comment:
Can we separate `DEFAULT_PEER_PROTECTION_FOR_LATENCY_AND_USEFULNESS` into two constants?